subsys/net/lib/ocpp/ocpp_j.h

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2023 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __OCPP_J_
+#define __OCPP_J_
+
+#define MAX_UID_LEN 38
+#define MAX_AUTH_REQ_LEN 32
+#define MAX_HEARTBEAT_REQ_LEN 2
+#define MAX_BOOTNOTIF_REQ_LEN 412
+#define MAX_METERVALUE_REQ_LEN 302
+#define MAX_STOPTRANSACTION_REQ_LEN 140
+#define MAX_STARTTRANSACTION_REQ_LEN 186
+#define MAX_UNLOCKCONNECTOR_RESP_LEN 24
+#define MAX_GETCONFIGURATION_RESP_LEN 92
+#define MAX_OCPP_RPC_MESSAGE_LEN 512
+
+#define BOOTNOTIF_REQ_FIELDS 2
+#define BOOTNOTIF_TOTAL_FIELDS 9
+
+#define STOP_TXN_REQ_FIELDS 3
+#define STOP_TXN_TOTAL_FIELDS 5
+
+#define START_TXN_REQ_FIELDS 4
+#define START_TXN_TOTAL_FIELDS 5
+
+#define GET_CFG_TOTAL_FIELDS 1
+
+#define SAMPLED_VALUE_REQ_FIELDS 2
+#define SAMPLED_VALUE_TOTAL_FIELDS 3
+
+struct authorize_payload {
```

Comment:
use generic `struct json_common_payload_field_str { char * val}` for `authorize_payload`,  `heartbeat_payload`, `ocpp_status_resp_msg`